packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -304,12 +327,21 @@ $form-input-focus--boxShadow              : 0 0 0 2px $form-input-focus;
         position: relative;
     }
 
-    .c-formField-field--defaultHeight {
-        @include rem(height, $form-input-height); //convert height to rem
+    .c-formField-field--small {
+        @include form-field-size($form-input-height-small, $form-input-padding-small); //convert height to rem
+    }
+
+    .c-formField-field--medium {
+        @include form-field-size($form-input-height-medium, $form-input-padding-medium); //convert height to rem
+    }
+
+    .c-formField-field--large {
+        @include form-field-size($form-input-height-large, $form-input-padding-large); //convert height to rem
     }
 
     .c-formField-field {
         width: 100%;
+        box-sizing: border-box;
```

Comment:
Does the element not get this as default? (it should come through as part of the reset stylesheet I think, unless it gets overidden)